logrotate.h

```diff
@@ -80,6 +80,9 @@ TAILQ_HEAD(logInfoHead, logInfo) logs;
 extern int numLogs;
 extern int debug;
 
+/* This is the definition. Note we have to carry the "2" within the declaration (logrotate.c), too! */
+extern const char * compress_cmd_list[][2];
```

Comment:
Please drop this.  We do not want locally used symbols to be globally available.